### ISSUE TITLE
Loss is NaN

### ISSUE THREAD

**Issue by phongnhhn92** [leaderj1001/Stand-Alone-Self-Attention#5]
Hello, 
I am testing your Resnet50 model with stem is True and at the first training step, my loss is NaN and the accuracy is decreasing? Is that a bug?
![image](https://user-images.githubusercontent.com/11288381/64538233-effb0680-d324-11e9-91e4-666ee7512dcf.png)

**Comment by leaderj1001** [comment]
Thanks for your comments.
I don't have enough GPUs. So, I couldn't experiments all of ResNet model. Maybe, you can reduce learning_rate. example) 0.01

Thank you !

**Comment by ksouvik52** [comment]
Hi, I am facing issues with the Resnet50 model training on CIFAR-10. Even with lr of 0.01 it's throwing Nan after around 10 epochs (suddenly), so, I am not quite sure how to train the resnet50 model. Hoping for a quick reply! Thanks.

Just as a note, the resnet38 and 26 did run  successfully without Nan.
